tfs/handler.py

```diff
@@ -218,7 +220,14 @@ def _insert_index_column(data_frame, save_index):
 
 
 def _get_headers_string(headers_dict, width) -> str:
-    return "\n".join(_get_header_line(name, headers_dict[name], width) for name in headers_dict)
```

Comment:
If you go with my suggestion above this will just return an empty string if headers_dict is empty